version/1/4/tw/code_of_conduct.txt

```diff
@@ -0,0 +1,49 @@
+貢獻者公約
+
+我們的承諾
+
+為了促進一個開放且受歡迎的環境，我們作為貢獻者和維護者保證，無論年齡、身高、殘疾、種族、性別認同和表達、無論年齡、體型、殘疾、種族、 經驗水平、國籍、個人表現、種族、宗教或性別認同和取向，在我們的專案以及社群的參與者都有不被騷擾的體驗。
```

Comment:
To me "受歡迎的環境" seems to mean an environment that is welcome by others (e.g. the community at large) rather than an environment that welcomes others.

Also, some of the "prohibited harassment topics" here have been duplicated.